_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -956,6 +956,11 @@ public function update_data( $request ) {
 					$updated = get_option( $option ) !== $value ? update_option( $option, (bool) $value ) : true;
 					break;
 
+				case 'disable_tracking':
```

Comment:
This is the same than above (and similar to the one below except for the !== ) so I think it's unnecessary to repeat all the code.